### DISCUSSION TITLE
Release v1.1.0.0

### DISCUSSION THREAD

**Discussion by jlanz** [dreamworksanimation/openmoonray#102] [Announcements]
#### moonray
* New extensions to MoonRay's Cryptomatte support to include options to output additional layers for each id/weight pair, including:
  - position data
  - normal data
  - beauty image
  - support for Cryptomatte data for each transmission depth as a separate layer
  - support for Cryptomatte data for each presence depth as a separate layer
* Added support for normal-oriented curves
* Triangulate concave ngons using the ear clipping method
* Updated SceneClass documentation

#### moonshine
* Removed ColorCorrectNukeMap
* Updated SceneClass documentation

#### moonshine_usd
* Added support for normal-oriented curves
* Updated SceneClass documentation

#### scene_rdl2
* Installs rdla_filter script
* Updated SceneClass documentation
* Updated SceneVariable documentation

## What's Changed

* Miscelleneaous updates for internal DWA builds
* Removed duplicate "The Bad Guys" reference in README.md [#60](https://github.com/dreamworksanimation/openmoonray/issues/60) [#63](https://github.com/dreamworksanimation/openmoonray/issues/63) [#73](https://github.com/dreamworksanimation/openmoonray/issues/73)
* Updated text and fixes link in CONTRIBUTING.md and ccla.md in all repositories

#### moonray
* Fixed moonray_bake per-part primitive attributes and improved performance
* Fixed homogenous volume scale factor
* Fixed vector mode closest aov filter
* Fixed volume flickering on different CPUs
* Fixed some black dots in Cryptomatte coverage
* Fixed depth buffer with presence in vector mode
* Fixed albedo AOV (and material AOVs in general) with presence in scalar mode

#### moonshine
* Fixed projection normal maps on deforming meshes

#### scene_rdl2
* Fixed enable_if UI metadata for local motion blur

## Other News

* We've released our "MoonRayWidget" model as seen in many of our documentation images. It's licensed under ASWF Digital Assets License v1.1 and can be found [here](https://docs.openmoonray.org/getting-started/test-scenes/)

<hr /><em>This discussion was created from the release <a href='https://github.com/dreamworksanimation/openmoonray/releases/tag/v1.1.0.0'>v1.1.0.0</a>.</em>

**Comment by Ponlets** [comment]
i am not a coder... is there a way to install this into Blender ? i would aim to use the latest version of Blender with this

**Reply by jmahovsky-dwa** [reply]
Unfortunately there is not yet a Blender plugin for MoonRay.
